### ISSUE TITLE
Not able to add second network interface

### ISSUE THREAD

**Issue by sheetaljoshi** [BBVA/kvm#28]
Hi, 

I have two docker networks created as explained in README. But when I follow the steps I don't see the second network getting added. It updates same eth0 with the last network connected. I have brwan1 and brlan1 networks created and I create container using  below command 

docker create                                 \
      --name gw-vm                     \
      -it  							\
      --privileged \
      --network=network1	                                 \
      -v /images/gw-vm.qcow2:/image/image \
      -e AUTO_ATTACH=yes 	\
      bbvainnotech/kvm:latest

and then run docker network connect network2 gw-vm && docker start gw-vm. But when I docker attach gw-vm and check for ifconfig I only see eth0 and it updated with brlan1 ip range but not two nics added.

Can you please suggest what might be wrong here?

**Comment by methadata** [comment]
How did you solve this? Was anything related with the container or a bug?

**Comment by sheetaljoshi** [comment]
Hi,

I was initially trying to just do docker run with one network interface and that erased the port information on my image, which is strange and hence network connect after run never added a new interface for guest VM, which seems ok by design. Because network connect just add interface at container level but running guest qemu command never gets added this new network interface.

And after docker run and docker network connect had this issue, I tried docker create, docker network connect and docker start on the same vm image which has already lot the port information , it never added a second interface, this is when I created this isuse.

And later I did build a clean qcow2 image which had two port information now, I did docker create , docker network connect and docker start. That has been able to retain both the ports on the VM and hence interfaces.

Thanks,
Sheetal
